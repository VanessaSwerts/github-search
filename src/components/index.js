import CustomHeader from './Header/Header'
import CustomFooter from './Footer/Footer'
import Error from './Error/Error'
import Loading from './Loading/Loading'

export {
  CustomHeader,
  CustomFooter,
  Error,
  Loading
}