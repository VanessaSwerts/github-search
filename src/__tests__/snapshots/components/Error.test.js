import React from 'react'
import { render } from '@testing-library/react'
import { Error } from '../../../components'

describe('Testes de Snapshot da Error', () => {

  test('Renderização do componente Error', () => {
    const { asFragment } = render(<Error error="Teste mensagem de erro!" />)

    expect(asFragment(<Error />)).toMatchSnapshot()
  })

}) 