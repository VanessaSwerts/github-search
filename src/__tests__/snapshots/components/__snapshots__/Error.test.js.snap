// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes de Snapshot da Error Renderização do componente Error 1`] = `
<DocumentFragment>
  <div
    class="card-error"
  >
    <i
      class="fas fa-exclamation-triangle"
    />
    <span
      data-testid="message-error"
    >
      Teste mensagem de erro!
    </span>
  </div>
</DocumentFragment>
`;
